### DISCUSSION TITLE
IPQos cs0 cs0

### DISCUSSION THREAD

**Discussion by sejtam** [outdoorbits/little-backup-box#218] [Ideas]
I found that on my LBB, an SSH connection started failing after a while, esp after being idle

https://omcar.medium.com/raspberry-pi-ssh-freezing-solution-2b503a6af3b0

suggests (and this worked for me on other RPis) to add

IPQos cs0 cs0

to /etc/ssh/sshd_config

I would suggest that that be made default for LBB too?

**Comment by outdoorbits** [comment]
Good point, I'll implement this in short.

**Comment by outdoorbits** [comment]
Thank you, your idea is implemented.
